transport/http.go

```diff
@@ -17,6 +17,23 @@ func writeResponse(w http.ResponseWriter, status int, data interface{}) {
 	w.WriteHeader(status)
 }
 
+func pingHandler(w http.ResponseWriter, r *http.Request) {
+	resp := map[string]string{
+		"status": "ok",
+	}
+	w.Header().Set("content-type", "application/json")
+	w.WriteHeader(200)
+	writeResponse(w, http.StatusOK, resp)
```

Comment:
This already handles writing the content-type header and status code. No need for the two lines above.